clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -2,17 +2,18 @@ package io.treeverse.clients
 
 import io.lakefs.clients.api
 import io.lakefs.clients.api.{ConfigApi, RetentionApi}
-import io.lakefs.clients.api.model.{
-  GarbageCollectionPrepareRequest,
-  GarbageCollectionPrepareResponse
-}
+import io.lakefs.clients.api.model.{Commit, GarbageCollectionPrepareRequest, GarbageCollectionPrepareResponse, GarbageCollectionRules, Ref, Repository, StorageConfig, StorageURI}
 import io.treeverse.clients.StorageClientType.StorageClientType
 import io.treeverse.clients.StorageUtils.{StorageTypeAzure, StorageTypeS3}
 import com.google.common.cache.{Cache, CacheBuilder, CacheLoader, LoadingCache}
+import dev.failsafe.function.CheckedSupplier
 import io.treeverse.clients.ApiClient.TIMEOUT_NOT_SET
 
 import java.net.URI
 import java.util.concurrent.{Callable, TimeUnit}
+import java.time.Duration
+import collection.JavaConverters._
+import dev.failsafe.{Failsafe, Policy, RetryPolicy}
```

Comment:
Nit: Let's keep all the `failsafe` imports together if possible!  Also, if we have a blank line at l. 13/11 then let's keep standard library (`java.*`, `collection.*`) below it and everything else above it.

Reply:
In IntelliJ: control + option + O